Public/Sdk/Public/Prelude/Prelude.IO.dsc

```diff
@@ -48,6 +48,12 @@ declare function globFolders(folder: Directory, pattern?: string, isRecursive?:
 declare function globR(folder: Directory, pattern?: string): File[];
 declare function globRecursively(folder: Directory, pattern?: string): File[];
 
+/**
+ * Unsafe functions to create output file artifact and exclusive output directory.
+ */
+declare function unsafe_outputFile(path: Path, rewriteCount?: number): File;
+declare function unsafe_exOutputDirectory(path: Path): Directory;
+

```

Comment:
Please place these in a namespace 'Unsafe' just so it is contained....

Reply:
I created `Prelude.Unsafe.dsc`

---
In reply to: [313613352](https://github.com/microsoft/BuildXL/pull/730#discussion_r313613352) [](ancestors = 313613352)